manifest_cultural/templates/index.html

```diff
@@ -79,10 +80,14 @@ <h1>Welcome to Events Selector</h1>
 
     <section id="select">
       <div class="main-container">
-        <form action="javascript:void(0);" onSubmit="getCheckedBoxes(this)" method="post">
+        <p><b><u>Select your desired events and press submit: </u></b></p>
+        <form action="javascript:void(0);" onSubmit="getCheckedBoxes(this) &
+                      autosize(document.getElementById('edit-area'));" method="post">
```

Comment:
don't you want to include this in the function also? and maybe rename the function since it does multiple things?